querylog/database_writer.go

```diff
@@ -113,7 +114,30 @@ func databaseMigration(db *gorm.DB) error {
 		}
 
 	case "postgres":
-		return db.Exec("ALTER TABLE " + tableName + " ADD column if not exists id serial primary key").Error
+		return db.Exec("ALTER TABLE " + tableName + " ADD column if not exists id bigserial primary key").Error
```

Comment:
Why did the type change from serial to bigserial?

Reply:
Depending on retention time I think is possible to reach the max value of serial. That's why I changed it to bigserial.